docs/locations.md

```diff
@@ -27,13 +41,23 @@ When you use the activate list, all other plugins that a user had loaded will be
 https://remix.ethereum.org/?#activate=solidity,solidityUnitTesting,udapp,defiexplorer
 ```
 
+Note: a plugin is called by its **name** in its profile.  There are three types of plugins - internally built plugins that are always loaded, internally built plugins that are loaded on demand and externally built plugins. 
```

Comment:
May be we can categorise them as:

1. Native Plugins

  -  Mandatory or Default Plugins
  -  Optional Plugins

2. External Plugins